engine/app/components/citizens_advice_components/textarea.rb

```diff
@@ -2,28 +2,38 @@
 
 module CitizensAdviceComponents
   class Textarea < Input
```

Comment:
Noting that having this subclass from `Input` is getting a bit unwieldy. We should probably separate these out.